vars/piperStageWrapper.groovy

```diff
@@ -188,3 +188,7 @@ private boolean isOldInterceptorInterfaceUsed(Script interceptor) {
     MetaMethod method = interceptor.metaClass.pickMethod("call", [Closure.class, String.class, Map.class, Map.class] as Class[])
     return method != null
 }
+
+private boolean allowExtensions(){
+    return env.PIPER_DISABLE_EXTENSIONS == null || Boolean.valueOf(env.PIPER_DISABLE_EXTENSIONS) == true
```

Comment:
I think, this does not work, if I set the env variable to `true`, the return value if `true `and it actually executes the code inside the if condition.